src/components/allReviews/ReviewFilters/SearchBox.jsx

```diff
@@ -0,0 +1,28 @@
+const SearchBox = ({ setSearch, Search }) => {
+    const handleSubmit = (e) => {
+        e.preventDefault();
+        let queryString = `${e.target[0].value}${e.target[1].value}`
+        setSearch(queryString)
+    }
+
+    return (<form autocomplete="off" onSubmit={(e) => handleSubmit(e)}>
+        <label htmlFor="newSearch"></label>
```

Comment:
This label tag is empty and also the htmlFor is not linking to any name properties in this component?